### PR TITLE
Thinking about mobile - A short post and video about our the Holiday Extras mobiletastic week.

### PR DESCRIPTION
**Pull request by jodiedoubleday**
### Blog title
Thinking about mobile

### Blog description
A short post and video about our the Holiday Extras mobiletastic week.

### Extra
As this post includes a video, i have added some styling to enable the youtube video to be responsive.

#### Author
- [x] I have checked for grammatical errors
- [x] I have checked for spelling errors
- [x] All information (technical or otherwise) is correct to my knowledge

#### Editor 1
- [x] I have checked for grammatical errors
- [x] I have checked for spelling errors
- [x] All information (technical or otherwise) is correct to my knowledge

#### Editor 2
- [x] I have checked for grammatical errors
- [x] I have checked for spelling errors
- [x] All information (technical or otherwise) is correct to my knowledge

### CODE REVIEW

File: _posts/2015-04-02-thinking-about-mobile.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+comments: true
+---
+
+We haven't yet had our [mobile moment](http://www.lukew.com/ff/entry.asp?1841]) at Holiday Extras but that hasn't stopped us as a tech team taking careful, considered approaches to everything we do on our websites, ensuring we always think mobile first.
+
+So in January we decided to share our knowledge by running a session, we lovingly called **mobiletastic week**.
+
+The aim was to get the rest of our in-house teams thinking about the power of mobile devices and what we can do as a company to use this power to benefit our customers.
+
+We recorded a summary of the events we put on, let us know what you think.
```
Comment (HarleyRowland) [inline _posts/2015-04-02-thinking-about-mobile.md]: Should this be "We recorded a summary of the events we put on. Let us know what you think."?

**Comment by HarleyRowland** [comment]
Taking a review

**Comment by lukefrake** [comment]
Taking a review

File: _posts/2015-04-02-thinking-about-mobile.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+author: jodie_doubleday
+tags: Mobile Video
+comments: true
+---
+
+We haven't yet had our [mobile moment](http://www.lukew.com/ff/entry.asp?1841]) at Holiday Extras but that hasn't stopped us as a tech team taking careful, considered approaches to everything we do on our websites, ensuring we always think mobile first.
+
+So in January we decided to share our knowledge by running a session, we lovingly called **mobiletastic week**.
+
+The aim was to get the rest of our in-house teams thinking about the power of mobile devices and what we can do as a company to use this power to benefit our customers.
```
Comment (lukefrake) [inline _posts/2015-04-02-thinking-about-mobile.md]: I think ' the rest of our in-house teams' makes it sound like a very us & them relationship. Maybe something more like "The aim was to get everybody thinking"?

File: _posts/2015-04-02-thinking-about-mobile.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+We recorded a summary of the events we put on, let us know what you think.
+
+<div class='embed-container'>
+  <iframe src='https://www.youtube.com/embed/nICDI6UZPNg?rel=0&amp;controls=0&amp;showinfo=0' frameborder='0' allowfullscreen></iframe>
```
Comment (lukefrake) [inline _posts/2015-04-02-thinking-about-mobile.md]: I think this should be without the HX strap line at the end, and from the HG youtube account (Which I am speaking to Gary about now). Does that sound like something we could do?

File: _posts/2015-04-02-thinking-about-mobile.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+layout: post
+title: Thinking about mobile
+date: 2015-04-02 08:20:00
+categories: Mobile
+author: jodie_doubleday
+tags: Mobile Video
+comments: true
+---
+
+We haven't yet had our [mobile moment](http://www.lukew.com/ff/entry.asp?1841]) at Holiday Extras but that hasn't stopped us as a tech team taking careful, considered approaches to everything we do on our websites, ensuring we always think mobile first.
```
Comment (lukefrake) [inline _posts/2015-04-02-thinking-about-mobile.md]: 'at Holiday Extras yet' ?

**Comment by jodiedoubleday** [comment]
I don't think the multimedia team have the capacity to edit the end of the video any time soon.
Want to get this out soon, so happy for it to be from a hungry geek youtube account if you can get it set up soon @lukefrake otherwise ill stick live and we can change the embed at a later date.
Let me know.

**Comment by HarleyRowland** [comment]
Looks good :+1:

**Comment by lukefrake** [comment]
I agree with you mate

**Pull request merged**